applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -300,6 +311,23 @@ def __main():
         default=False,
         help="Set when using nRF21540 for extra TX power",
         )
+    parser.add_argument(
+        "-u",
+        "--user_bt_name",
+        action="store_true",
+        dest="user_bt_name",
+        default=False,
+        help="Set to generate a user specific Bluetooth device name. Note that this will put the user name on air in clear text.",
+        )
+    parser.add_argument(
+        # should only be used when --user_bt_name is not used
+        "-cn",
+        "--custom_bt_name",
+        nargs='*',
```

Comment:
Why do we need to allow for multiple arguments? Shouldn't this argument be given only once?

Reply:
It is not necessary. But now you can write your name with spaces between first and last name and it will set the BT dev name to "MARIUS_VÅRDAL", for example